### ISSUE TITLE
The user gitpod does not have sudo permission

### ISSUE THREAD

**Issue by dclong** [gitpod-io/gitpod#501]
**Describe the bug**
The user `gitpod` does not have sudo permission even though the command `id` shows that it belongs to the `sudo` group. 

**To Reproduce**
Run `sudo apt-get update`

**Expected behavior**
The user `gitpod` should have sudo permission since it's in the sudo group.

**Comment by svenefftinge** [comment]
Agreed! We are working on allowing sudo/root. ATM it is disabled for security reasons.
Please subscribe to #39.

**Comment by kevinhq** [comment]
Any update on this?  I got into problem installing ruby version with rvm and it looks like it's a problem with sudo permission.

**Comment by sachinsh76** [comment]
any updates on this?

**Comment by Kreyren** [comment]
@sachinsh76 The issue is being tracked #39 

FWIW from my understanding TypeFox doesn't want to provide the root access for security reasons.

I've refactored whole https://github.com/gitpod-io/workspace-images to allow all linux-based distros, stub for other kernels and provided research from mensioned concerns about possible solutions in https://github.com/gitpod-io/gitpod/issues/1251#issuecomment-589881514 and https://github.com/docker/compose/issues/7244#issuecomment-625834398 as an attempt to contribute in solution (to which i didn't get a response for almost a month now and eventually gave up on it..) of this issue, but it seems to be blocked by https://github.com/kubernetes/kubernetes/pull/64005 which is on hold due to the https://github.com/kubernetes/enhancements/issues/127#issuecomment-465283951, but i wasn't briefed about any of the issues to be certain (and to possibly contribute more) so for now i am implementing following https://github.com/Krey-s-Torter/torter/blob/master/gitpod/scripts/root-access.sh to avoid confused and flustrated contributors for projects that depends on these.

Hopefully @svenefftinge is going to provide more info when TypeFox considers it's being safe/sane to do so assuming current kubernetes limitations(?).
